src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,75 @@
 package mate.jdbc;
 
+import java.util.List;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
-        // test your code here
+        ManufacturerService manufacturerService
+                = (ManufacturerService) injector.getInstance(ManufacturerService.class);
+        System.out.println("Manufacturer Service testing:");
+        exampleOfUsageManufacturerService(manufacturerService);
```

Comment:
Remember about method naming convention.
```suggestion
        runExampleOfUsageManufacturerService(manufacturerService);
```